css-scroll-snap-2/Overview.bs

```diff
@@ -193,26 +193,117 @@ The 'scroll-start' property {#scroll-start}
 The 'scroll-start-target' property {#scroll-start-target}
 -------------------------------------------
 
-  <pre class="propdef shorthand">
-  Name: scroll-start-target
-  Value: [ none | auto ]{1,2}
-  </pre>
-
-  This property is a shorthand property that sets all of the scroll-start-target-* longhands in one declaration.
-  The first value defines the scroll starting point in the block axis,
-  the second sets it in the inline axis.
-  If the second value is omitted, it defaults to ''none''.
-
-  Values are defined as follows:
-
-  <dl dfn-type=value dfn-for="scroll-start-target, scroll-start-target-x, scroll-start-target-y, scroll-start-target-block, scroll-start-target-inline">
-    <dt><dfn>none</dfn>
-    <dd>Element is not a ''scroll-start-target''.
-    <dt><dfn>auto</dfn>
-    <dd>Element is used to calculate the ''scroll-start'' position, 
-    taking into account ''scroll-padding'' or ''scroll-margin'' ,
-    same as a ''scroll-snap'' target.
-  </dl>
+	<h4 id="initial-scroll-target">Initial scroll target</h4>
+
+		The <dfn>initial scroll target</dfn> of a <a>scroll container</a> |scrollcontainer|
+		with respect to a particular axis is an element or pseudo-element whose {{scroll-start-target}}
+		property for that axis is non-''none'' and whose nearest <a>scroll container</a> is
+		|scrollcontainer|. Where multiple such elements or pseudo-elements exist, user-agents
+		should select the one which comes first in <a href="https://www.w3.org/TR/dom/#concept-tree-order">tree order</a>. Where no such element
+		or pseudo-element exists, |scrollcontainer|’s <a>initial scroll target</a> in
+		that axis is null.
```

Comment:
I think you'll also need to define the interaction with the [try to scroll to the fragment](https://html.spec.whatwg.org/#try-to-scroll-to-the-fragment) step of navigation. In particular, this is the part that deals with cases where the element to scroll to is not in the initial render. I think we'll want this to also detect cases where a scroll-start-target was not originally present but shows up in the document.

Reply:
I think this would still be covered by what you pointed out earlier: scroll-start-target only determines what the initial scroll position should be. So it's implied that a scroll-start-target that shows up later only changes the initial scroll position of some affected scroller but not (necessarily) its current scroll position, similar to place-content affecting the initial scroll position. Or perhaps you mean that all this is somewhat too implicit and needs to be called out directly?